pegasus/sites.v3/code.org/views/mobile_footer_responsive.haml

```diff
@@ -1,11 +1,48 @@
--# an utra-simple mobile footer
+%link{:href=>'/css/mobile-footer.css', :rel=>'stylesheet'}
+%script{src: '/js/mobile-footer.js'}
 
 %footer#pagefooter
-  .content{:style=>'width:100%'}
-    .left{:style=>'float:left'}
-      - if request.language == "en"
-        %a.whitefooterlink{:href=>"/privacy", :style=>"color: #ffa400;"} Privacy Policy
-      %br
-      !="&copy; Code.org, #{Time.now.year}"
-    .right{:style=>'float:right'}
```

Comment:
When I try to change the language, the page automatically switches back to English.